consensus/src/network/hub.rs

```diff
@@ -0,0 +1,105 @@
+use crate::{
+    alerts::AlertMessage,
+    member::UnitMessage,
+    network::{NetworkData, NetworkDataInner},
+    Data, Hasher, Network, PartialMultisignature, Receiver, Recipient, Sender, Signature,
+    Terminator,
+};
+use futures::{FutureExt, StreamExt};
+use log::{debug, error, warn};
+
+pub struct Hub<
```

Comment:
Overall, the change looks good, but I just have one question: why rename `NetworkHub` -> `Hub`? I liked the former better; it was the more specific name (and `Hub` still has `network` as a field).

Reply:
Oh, it was because it's `network::Hub` anyway, I prefer renaming this on import to having `network::NetworkHub`, but this is a small preference, I can rename it back.

Reply:
You're right that module gives proper naming anyway, so it's fine to leave it as it is.